packages/web/components/complex/paginated-table.tsx

```diff
@@ -79,25 +78,31 @@ export const PaginatedTable = ({
         {virtualRows.map((virtualRow) => {
           const row = rows[virtualRow.index] as Row<ObservablePoolWithMetric>;
           return (
-            <Link
+            <a
               key={row.original.queryPool.id}
               href={getPoolLink(row.original.queryPool)}
-              passHref
-              legacyBehavior
+              style={{
```

Comment:
we try to avoid inline styles, and instead use [tailwind](https://tailwindcss.com/) classes